packages/bundlemon/src/main/outputs/outputs/custom.ts

```diff
@@ -0,0 +1,50 @@
+import * as yup from 'yup';
+import path from 'path';
+import { Report } from 'bundlemon-utils';
+import { createLogger } from '../../../common/logger';
+import { validateYup } from '../../utils/validationUtils';
+import type { Output, OutputInstance } from '../types';
+
+const NAME = 'custom';
+
+const logger = createLogger(`${NAME} output`);
+
+interface CustomOutputOptions {
+  path?: string;
+}
+interface NormalizedCustomOutputOptions {
+  path: string;
+}
+
+export type CustomOutputFunction = (results: Report) => any;
+
+function validateOptions(options: unknown): NormalizedCustomOutputOptions {
+  const schema: yup.SchemaOf<CustomOutputOptions, CustomOutputOptions> = yup.object().required().shape({
+    path: yup.string().required(),
+  });
+
+  const normalizedOptions = validateYup(schema, options, `${NAME} output`);
+  if (normalizedOptions === undefined) {
+    throw new Error(`validation error in output "${NAME}" options`);
+  }
+  return normalizedOptions as NormalizedCustomOutputOptions;
```

Comment:
Just for clarity: I hate this hack :sob:

Reply:
yeah, not sure what do to with that, will try to fix it later